addons/event/views/event_registration_views.xml

```diff
@@ -209,7 +209,7 @@
         </record>
 
         <record model="ir.ui.view" id="view_registration_search">
-            <field name="name">event.registration.search</field>
+            <field name="name">event.registration.search.specific.to.an.event</field>
```

Comment:
I think I would do the inverse move: keep the default view as it is (that way we don't break anything in custom modules) and define an ``event_registration_view_search_event_specific`` that inherit from the generic one and remove the event_id fields (and set it as less important, for example priority 32). Then I would use that specific one in the ``act_event_registration_stat_from_event`` .

That way for anything else that the new action, nothing changes and we keep behavior as previous unless specifically coming from an event.